cinder_service_check.py

```diff
@@ -53,6 +53,12 @@ def check(auth_ref, args):
 
     services = r.json()['services']
 
+    # We need to match against a host of X and X@lvm (or whatever backend)
+    if args.host:
+        backend = ''.join((args.host, '@'))
```

Comment:
```
~ python -m timeit -n 1000000 '"".join(("foo", "bar"))'
1000000 loops, best of 3: 0.104 usec per loop
~ python -m timeit -n 1000000 '"foo" + "bar"'
1000000 loops, best of 3: 0.0172 usec per loop
```

There is a difference of 10x performance wise. It isn't significant here, but consistency is the hobgoblin of foolish minds and we should choose the best solution, not the most consistent.